<?xml version="1.0" encoding="UTF-8"?>
<!--
FSECplotter2 - The interactive plotting application for FSEC.

Copyright 2015-2017, TaizoAyase, tikuta, biochem-fan

This file is part of FSECplotter2.

FSECplotter2 is free software: you can redistribute it and/or modify
it under the terms of the GNU General Public License as published by
the Free Software Foundation, either version 2 of the License, or
(at your option) any later version.

This program is distributed in the hope that it will be useful,
but WITHOUT ANY WARRANTY; without even the implied warranty of
MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
GNU General Public License for more details.

You should have received a copy of the GNU General Public License
along with this program.  If not, see <http://www.gnu.org/licenses/>.
-->
<ui version="4.0">
 <class>PreferenceDialog</class>
 <widget class="QDialog" name="PreferenceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="default_tab">
      <attribute name="title">
       <string>Default Parameters</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QGroupBox" name="detector_channel_group">
         <property name="title">
          <string>Detector / Channel parameters</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Default Detector:</string>
              </property>
              <property name="buddy">
               <cstring>detector_comboBox</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="detector_comboBox">
              <property name="currentText">
               <string notr="true">B</string>
              </property>
              <property name="currentIndex">
               <number>1</number>
              </property>
              <item>
               <property name="text">
                <string notr="true">A</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string notr="true">B</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Default Channel:</string>
              </property>
              <property name="buddy">
               <cstring>channel_spinBox</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="channel_spinBox">
              <property name="maximum">
               <number>10</number>
              </property>
              <property name="value">
               <number>1</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="flowrate_group">
         <property name="title">
          <string>Flow rate</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Default value when failed to get flow rate from logfile.</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Default flow rate:</string>
              </property>
              <property name="buddy">
               <cstring>flowrate_spinBox</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="flowrate_spinBox">
              <property name="decimals">
               <number>2</number>
              </property>
              <property name="maximum">
               <double>25.000000000000000</double>
              </property>
              <property name="value">
               <double>10.000000000000000</double>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>mL/min</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="plot_params">
         <property name="title">
          <string>Plot params</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_10">
            <item>
             <widget class="QLabel" name="label_16">
              <property name="text">
               <string>x-min default: </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="x_min_lineEdit">
              <property name="text">
               <string>0</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_17">
              <property name="text">
               <string>[mL/min]</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_11">
            <item>
             <widget class="QLabel" name="label_18">
              <property name="text">
               <string>x-max default:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="x_max_lineEdit">
              <property name="text">
               <string>30</string>
              </property>
              <property name="clearButtonEnabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_19">
              <property name="text">
               <string>[mL/min]</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>Default line width:</string>
              </property>
              <property name="buddy">
               <cstring>linewidth_spinBox</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="linewidth_spinBox">
              <property name="decimals">
               <number>1</number>
              </property>
              <property name="singleStep">
               <double>0.500000000000000</double>
              </property>
              <property name="value">
               <double>1.000000000000000</double>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>pt</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_12">
            <item>
             <widget class="QLabel" name="label_15">
              <property name="text">
               <string>x-axis:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="x_axis_comboBox">
              <property name="currentIndex">
               <number>0</number>
              </property>
              <item>
               <property name="text">
                <string>Volume [mL]</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Time [min]</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>157</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="figurecanvas_tab">
      <attribute name="title">
       <string>Figure Canvas Setting</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <item>
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>These parameters are enabled when app is restared.</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Default figure dpi: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="figure_dpi_lineEdit">
           <property name="text">
            <string>100</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>60</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="use_seaborn_checkBox">
         <property name="text">
          <string>use seaborn package</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="seaborn_style_groupBox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="title">
          <string>seaborn style setting</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string>Style:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="sns_style_comboBox">
              <item>
               <property name="text">
                <string>DarkGrid</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>WhiteGrid</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Dark (no grid)</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>White (no grid)</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>ticks</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <widget class="QLabel" name="label_14">
              <property name="text">
               <string>Context:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="sns_context_comboBox">
              <item>
               <property name="text">
                <string>None</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>paper</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>notebook</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>talk</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>poster</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>204</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="advanced_tab">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <attribute name="title">
       <string>Advanced</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="ts_group">
         <property name="title">
          <string>FSEC-TS parameter</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Initial value for curve sigmoid fitting of FSEC-TS tool. </string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Gain:</string>
              </property>
              <property name="buddy">
               <cstring>ts_gain_lineEdit</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="ts_gain_lineEdit">
              <property name="text">
               <string>1.0</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="clearButtonEnabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Tm:  </string>
              </property>
              <property name="buddy">
               <cstring>ts_tm_lineEdit</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="ts_tm_lineEdit">
              <property name="text">
               <string>50</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>238</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>use_seaborn_checkBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>seaborn_style_groupBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>62</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>70</x>
     <y>188</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
